org.mwc.cmap.core/src/org/mwc/cmap/core/property_support/TagListHelper.java

```diff
@@ -35,7 +36,21 @@ public TagListHelper(final String[] theTags,
 
 	public CellEditor getCellEditorFor(final Composite parent)
 	{
-		return new ComboBoxCellEditor(parent, _theTags);
+		return new ComboBoxCellEditor(parent, _theTags, SWT.READ_ONLY){

```

Comment:
Hmm, it seems strange that we show the Combo Box in SWT.READ_ONLY mode.  I can see it's working, but could you explain why it's READ_ONLY?